pkg/kubelet/container/runtime.go

```diff
@@ -52,6 +52,10 @@ type Runtime interface {
 	// TODO(yifan): Pull/Remove images
 }
 
+type ContainerRunner interface {
```

Comment:
@yifan-gu @vmarmol Eventually this method should be in the `container.Runtime` interface.  I can add it in this change, if you want, but I would like a narrow interface so we can easily consume this in the kubelet via that narrow interface until we're ready to switch to `container.Runtime`.